### PR TITLE
fixed numpad return not applying indents

### PR DESCRIPTION
**Pull request by d3rrial**
I hope this is more inline with what was intended :)

### CODE REVIEW

**Comment by eidheim** [comment]
Thank you! I'll look at this Tomorrow morning.

**Comment by zalox** [comment]
@eidheim Do any of these need attention?
https://github.com/cppit/jucipp/search?utf8=%E2%9C%93&q=GDK_KEY_Return

**Comment by eidheim** [comment]
@zalox Yes, most likely.

**Comment by d3rrial** [comment]
Shall I change those the same way I changed them in the pull request? Saves you the effort.

**Comment by eidheim** [comment]
Sure, that would have been great:)

**Comment by d3rrial** [comment]
I feel the need to mention that I replaced them as-is. I didn't pay attention if there was a place where GDK_KEY_Return was valid while GDK_KEY_KP_Enter was supposed to be invalid.

**Comment by eidheim** [comment]
No worries, we'll look through it before merging:) Thank you again!

File: src/source_spellcheck.cc [hunk 1]
```diff
@@ -56,10 +56,10 @@ Source::SpellCheckView::SpellCheckView() : Gsv::View() {
       if(backward_success) {
         if(last_keyval==GDK_KEY_BackSpace && !is_word_iter(iter) && iter.forward_char()) {} //backspace fix
         if((spellcheck_all && !get_source_buffer()->iter_has_context_class(context_iter, "no-spell-check")) || get_source_buffer()->iter_has_context_class(context_iter, "comment") || get_source_buffer()->iter_has_context_class(context_iter, "string")) {
-          if(!is_word_iter(iter) || last_keyval==GDK_KEY_Return) { //Might have used space or - to split two words
+          if(!is_word_iter(iter) || (last_keyval==GDK_KEY_Return || last_keyval==GDK_KEY_KP_Enter)) { //Might have used space or - to split two words
```
Comment (d3rrial) [inline src/source_spellcheck.cc]: I know the parenthesis here are superfluous logic-wise, but I added them regardless to mark that they belong together. iirc the compiler should optimize this away, but I don't know for sure.

Comment (zalox) [inline src/source_spellcheck.cc]: As long as it provides a feature I'm all for it! To me, readable code is a feature ;)

**Pull request merged**